### PR TITLE
Rotation between

### PR DESCRIPTION
**Pull request by Atlante45**
glm already has a way to get a quaternion from 2 vectors, and I'm sure their version is way more optimized than ours.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: e4897a8de77345e08800946c3433f3115cd32cf1 available at: https://gist.github.com/3769962301a6413eb01d<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6373.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6373.dmg

File: libraries/shared/src/GLMHelpers.cpp [hunk 1]
```diff
@@ -201,30 +201,7 @@ float angleBetween(const glm::vec3& v1, const glm::vec3& v2) {
...
-        // It is possible for axis to be nan even when angle is not less than EPSILON.
-        // For example when angle is small but not tiny but v1 and v2 and have very short lengths.
-        if (glm::isnan(glm::dot(axis, axis))) {
-            // set angle and axis to values that will generate an identity rotation
-            angle = 0.0f;
-            axis = glm::vec3(1.0f, 0.0f, 0.0f);
-        }
-    }
-    return glm::angleAxis(angle, axis);
+    return glm::quat(glm::normalize(v1), glm::normalize(v2));
```
Comment (ZappoMan) [inline libraries/shared/src/GLMHelpers.cpp]: wouldn't it be more clear to use this form:

```
detail::tquat<T, P> glm::rotation	(	detail::tvec3< T, P > const & 	orig,
detail::tvec3< T, P > const & 	dest 
)		
Compute the rotation between two vectors.
param orig vector, needs to be normalized param dest vector, needs to be normalized
```

Comment (Atlante45) [inline libraries/shared/src/GLMHelpers.cpp]: Indeed it would, I didn't even know they had that one!

**Comment by hifi-gustavo** [comment]
Build console output for commit: 118d05d824935f6dfd067c41e2b31667fe6b0227 available at: https://gist.github.com/3769962301a6413eb01d<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6373.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6373.dmg

**Comment by imgntn** [comment]
my bow and arrow work in this PR :+1:

**Pull request merged**